bundles/nl.esi.comma.expressions/src/nl/esi/comma/expressions/validation/ExpressionValidator.xtend

```diff
@@ -203,6 +203,13 @@ class ExpressionValidator extends AbstractExpressionValidator {
 	@Check
 	def checkTypingExpression(Expression e){
 		switch(e){
+		    ExpressionEqual:{
+		        val leftType = e.left.typeOf
+		        val rightType = e.right.typeOf
+                if(!leftType.synonym(rightType)){
```

Comment:
Why did you omit the null check? (see https://github.com/dbera/OfflineMBT/blob/main/bundles/nl.esi.comma.expressions/src/nl/esi/comma/expressions/validation/ExpressionValidator.xtend#L250) This might happen while editing pspec files, currently possibly leading to NullPointerExceptions.